docs/generator.js

```diff
@@ -311,24 +330,11 @@ function inputVal () {
 }
 
 function getMousePos(canvas, quadrant, event) {
-    let rect = canvas.getBoundingClientRect();
-    let x, y;
-    if (quadrant === "1") {
-        x = Math.round((event.clientX - rect.left) / 2);
-        y = Math.round(math.abs(event.clientY - rect.bottom) * 5);
-    } else if (quadrant === "2") {
-        x = Math.round((event.clientX - rect.right) / 2);
-        y = Math.round(math.abs(event.clientY - rect.bottom) * 5);
-    } else if (quadrant === "3") {
-        x = Math.round((event.clientX - rect.right) / 2);
-        y = -Math.round(math.abs(event.clientY - rect.top) * 5);
-    } else if (quadrant === "4") {
-        x = Math.round((event.clientX - rect.left) / 2);
-        y = -Math.round(math.abs(event.clientY - rect.top) * 5);
-    }
+    const rect = canvas.getBoundingClientRect();
+    const curQuad = mouseQuadrants(rect)[quadrant];
     mousePos = {
-        x: x,
-        y: y
+        x: Math.round((event.clientX - curQaud.x) / 2),
```

Comment:
Slight typo here... you have `curQaud` instead of `curQuad`

Could you fix this and then request a merge? Thanks!